tests/tracing/test_utils.py

```diff
@@ -54,6 +56,62 @@ def test_maybe_get_request_id():
             maybe_get_request_id(is_evaluate=True)
 
 
+@pytest.mark.parametrize(
+    ("field", "expected"),
+    [
+        # no dot
+        ("span.inputs", _ParsedField("span", "inputs", None)),
```

Comment:
Backticks can appear in the current implementation so ``span`name.inputs`` is a valid extract field.